tests/data_validation/great_expectations/checkpoints/description_validation.yml

```diff
@@ -3,7 +3,7 @@ config_version: 1.0
 template_name:
 module_name: great_expectations.checkpoint
 class_name: Checkpoint
-run_name_template: 'newsletter automation-description validation'
+run_name_template: 'Ready Automation Article with Title and Description'
```

Comment:
This is not applicable for Automation Article alone but all Articles. So need to name the template accordingly. Also it is for Description validation whether it is provided or not.

Reply:
Updated the run_name_template with All Articles Description Validation